network/p2p/libp2pUtils.go

```diff
@@ -162,18 +162,6 @@ func IPPortFromMultiAddress(addrs ...multiaddr.Multiaddr) (string, string, error
 	return "", "", fmt.Errorf("ip address or hostname not found")
 }
 
-// FlowProtocolID creates a libp2p protocol ID namespaced to a particular
-// spork. This prevents cross-talk between nodes from different sporks.
-func FlowProtocolID(sporkID flow.Identifier) protocol.ID {
-	return protocol.ID(FlowLibP2POneToOneProtocolIDPrefix + sporkID.String())
-}
-
-// generatePingProtocolID creates a libp2p protocol ID namespaced to a particular
-// spork. This prevents cross-talk between nodes from different sporks.
-func generatePingProtocolID(sporkID flow.Identifier) protocol.ID {
-	return protocol.ID(FlowLibP2PPingProtocolPrefix + sporkID.String())
-}
-
```

Comment:
We use these protocol ids mostly (and almost solely) for unicast stream management. Hence, moved them to `p2p/unicast` to avoid import-cycle.